tstrait/trait_model.py

```diff
@@ -186,6 +186,10 @@ class TraitModelFixed(TraitModel):
     ----------
     value : float
         Value of the simulated effect size.
+    negative : bool, default False
```

Comment:
I find ``negative`` a bit of a weird name. How about ``random_sign=False``?

Reply:
I agree with that!